src/components/UI/UFormButton/UFormButton.tsx

```diff
@@ -1,16 +1,20 @@
+export enum ButtonType {
+  SUBMIT = 'submit',
+  BUTTON = 'button',
+}
+
 interface IFormButton {
   text: string;
-  onClick: () => void;
```

Comment:
onClick handler needed if button is type of "button", so you don't need to remove it